features/lockscreen/impl/src/main/res/values/localazy.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+  <plurals name="screen_app_lock_subtitle_wrong_pin">
+    <item quantity="one">"Wrong PIN. You have %1$d more chance"</item>
```

Comment:
@amshakal do we want to put something more alarming here? We had this in EA currently, for the last attempt:

```xml
<string name="wrong_pin_message_last_remaining_attempt">Warning! Last remaining attempt before logout!</string>
```

Reply:
Good idea! Ill craft something a bit more alarming, thank you!